### PR TITLE
RP103

### PR DESCRIPTION
**Pull request by ilaredo**
## Description
Moved RP103 to new validate

### CODE REVIEW

File: .changelog/4349.yml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+changes:
+  - description: Added RP103 to the new validation format.
```
Comment (YuvHayun) [inline .changelog/4349.yml]:
```suggestion
  - description: Added RP103 to the new validation format. The validation <what it does>.
```

File: .changelog/4361.yml [hunk 2]
```diff
@@ -0,0 +1,4 @@
+changes:
+  - description: Added RP103 to the new validation format.
```
Comment (YuvHayun) [inline .changelog/4361.yml:2]:
```suggestion
  - description: Added RP103 to the new validation format. The validation <what id does>.
```

**Review by YuvHayun — APPROVED** [review]
Nice, minor comments.

File: demisto_sdk/commands/validate/tests/RP_validatiors_test.py [hunk 3]
```diff
@@ -55,3 +58,49 @@ def test_ExpirationFieldIsNumericValidator_is_valid(
             for result, expected_msg in zip(results, expected_msgs)
         ]
     )
+
+
+@pytest.mark.parametrize(
```
Comment (dantavori) [inline demisto_sdk/commands/validate/tests/RP_validatiors_test.py:63]: as you might have heard - avoid using pytest.mark.parametrize
instead, split into different test methods, each one with more specific naming and docstring based on the test case.
Another option: just call the is_valid() multiple times in the same method. Still, I'd split into "valid" and "invalid" test methods.

Generally, this decision was made collectively in the team a few months ago, and the goal is for better readability and maintenance for the tests.
there are some cases where it is ok to use parametrize, e.g. - in very very simple tests with up to 2 arguments, that shouldn't be modified with more test-cases over the time. When you think this is the case, consult with Eyal/me if you want to use it.

File: demisto_sdk/commands/common/constants.py [hunk 4]
```diff
@@ -821,6 +821,8 @@ class FileType(StrEnum):
 
 LAYOUT_FILE_NAME__REGEX = r"layout.*\.json"
 
+VALID_INDICATOR_TYPE = "^[A-Za-z0-9_& ]*$"
```
Comment (dantavori) [inline demisto_sdk/commands/common/constants.py:824]:
```suggestion
VALID_INDICATOR_TYPE_REGEX = "^[A-Za-z0-9_& ]*$"
```

File: demisto_sdk/commands/validate/validators/RP_validators/RP103_is_valid_indicator_type_id.py [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+ContentTypes = IndicatorType
+
+
+# Valid indicator type can include letters, numbers whitespaces, ampersands and underscores.
+
+
+class IsValidIndicatorTypeId(BaseValidator[ContentTypes]):
+    error_code = "RP103"
+    description = "Validate that the 'id' field of indicator type has valid value."
+    error_message = "id field contain invalid value."
```
Comment (dantavori) [inline demisto_sdk/commands/validate/validators/RP_validators/RP103_is_valid_indicator_type_id.py:23]:
```suggestion
    error_message = "The `id` field must consist of alphanumeric characters (A-Z, a-z, 0-9), underscores (_), and ampersands (&) only."
```

**Review by dantavori — COMMENTED** [review]
nice! see notes

**Review by dantavori — APPROVED** [review]

**Pull request merged**